test/site_encrypt_test.exs

```diff
@@ -30,12 +30,10 @@ for {input, index} <- Enum.with_index(inputs),
     setup_all do
       start_supervised!({
         TestEndpoint,
-        endpoint_opts: [
```

Comment:
This caused opts to be

```elixir
[
  endpoint: SiteEncrypt.Phoenix.Endpoint.Cowboy2Adapter.NativeTest.TestEndpoint,
  endpoint_opts: [
    endpoint_opts: [
      adapter: Phoenix.Endpoint.Cowboy2Adapter,
      http: [port: 40000],
      https: [port: 40001],
      url: [scheme: "https", host: "localhost", port: 40001]
    ]
  ]
]
```

Reply:
Not sure though if it's a test issue or a configuration issue.

Reply:
It was introduced in a680884f09d0aa6038cfdc297b6cfadfdc2fe0a6. This only affected tests though.